libpod/image/pull.go

```diff
@@ -0,0 +1,252 @@
+package image
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	cp "github.com/containers/image/copy"
+	"github.com/containers/image/directory"
+	"github.com/containers/image/docker"
+	dockerarchive "github.com/containers/image/docker/archive"
+	"github.com/containers/image/docker/tarfile"
+	ociarchive "github.com/containers/image/oci/archive"
+	"github.com/containers/image/pkg/sysregistries"
+	is "github.com/containers/image/storage"
+	"github.com/containers/image/tarball"
+	"github.com/containers/image/transports/alltransports"
+	"github.com/containers/image/types"
+	"github.com/pkg/errors"
+)
+
+var (
+	// DockerArchive is the transport we prepend to an image name
+	// when saving to docker-archive
+	DockerArchive = dockerarchive.Transport.Name()
+	// OCIArchive is the transport we prepend to an image name
+	// when saving to oci-archive
+	OCIArchive = ociarchive.Transport.Name()
+	// DirTransport is the transport for pushing and pulling
+	// images to and from a directory
+	DirTransport = directory.Transport.Name()
+	// TransportNames are the supported transports in string form
+	TransportNames = [...]string{DefaultTransport, DockerArchive, OCIArchive, "ostree:", "dir:"}
```

Comment:
This doesn't include Docker and TarballTransport

Reply:
I believe this is correct.  One cannot pull from tarball and the default is docker

Reply:
Can we set DefaultTransport = Docker then, instead of a hardcoded string?